Libraries/ReactNative/AppContainer.js

```diff
@@ -11,15 +11,13 @@
 'use strict';
 
 const EmitterSubscription = require('../vendor/emitter/EmitterSubscription');
-const PropTypes = require('prop-types');
 const RCTDeviceEventEmitter = require('../EventEmitter/RCTDeviceEventEmitter');
+const PropTypes = require('prop-types');
 const React = require('react');
```

Comment:
If you change this to 

Import * as React from ‘react’;

Then you can use React.Context below instead of React$Context.